components/binding-usage-controller/examples/README.md

```diff
@@ -115,3 +115,19 @@ To perform a clean-up, remove the Namespace:
 ```bash
 kubectl delete ns $namespace
 ```
+
+## Pluggable SBU
+
+The feature flag **APP_PLUGGABLE_SBU=true** change Binding Usage Controller logic. The following scenario shows, how to test it:
```

Comment:
The feature flag **APP_PLUGGABLE_SBU=true** changes Binding Usage Controller logic. The following scenario shows how to test it: